index.html

```diff
@@ -49,10 +49,16 @@ <h1 class="title">My todos</h1>
         </div>
         <div class="clear-btn-container">
           <button id="clearCheckedTodosBtn" class="clear-btn">
-            Clear Checked
+            Clear completed
           </button>
         </div>
       </div>
     </div>
+    <footer>
+      <p>
+        <br />Double-click to edit a todo <br />Credits: Christoph Burgdorf,
+        Eric Bidelman, Jacob Mumm and Igor Minar <br />Part of TodoMVC
+      </p>
```

Comment:
why would you need this?

Reply:
That's the footer of the demo, isn't it?

https://todomvc.com/examples/angularjs/#/